doc/kak.1

```diff
@@ -1,251 +1,282 @@
-source code files:
-
-.nf
-.RS
-kak \-f "ggO// kak: tabstop=8<esc>" *.c
-.RE
-.fi
-
-.SH SEE ALSO
-
-.BR vi (1),
-.BR vim (1),
-.BR sam (1plan9)
+.Dd June 15, 2021
+.Dt KAK 1
+.Os
+.Sh NAME
+.Nm kak
+.Nd a vim-inspired, selection oriented code editor
+.
+.Sh SYNOPSIS
+.Nm
+.Op Fl d
+.Op Fl n
+.Op Fl ro
+.Op Fl c Ar session_id | Fl s Ar session_id
+.Op Fl ui Ar ui_type
+.Op Fl e Ar command
+.Op Fl E Ar command
+.Op Ar +line Ns Oo Ar :column Oc | +:
+.Op Ar file ...
+.
+.Nm
+.Fl f Ar keys
+.Op Fl q
+.Op Fl i Ar suffix
+.Op Ar file ...
+.
+.Nm
+.Fl p Ar session_id
+.
+.Nm
+.Fl l
+.Nm
+.Fl clear
+.
+.Nm
+.Fl version
+.
+.Nm
+.Fl help
+.
+.Sh DESCRIPTION
+.Sy Kakoune
+is a code editor heavily inspired by
+.Xr vim 1
+and
+.Xr vi 1 .
+As such, most of its commands are similar to
+.Xr vi 1 Ap s,
+sharing its
+.Dq keystrokes as a text editing language
+model.
+.Pp
+.Sy Kakoune
+operates in two modes, normal and insertion.
+In insertion mode, keys are directly inserted into the current buffer.
+In normal mode, keys are used to manipulate the current selection and to
+enter insertion mode.
+.Pp
+.Sy Kakoune
+has a strong focus on interactivity.
+Most commands provide immediate and incremental results, while still
+being competitive in keystroke count with vim.
+.Pp
+.Sy Kakoune
+works on selections, which are oriented, inclusive ranges of characters.
+Selections have an anchor and a cursor.
+Most commands move both of them except when extending selection, where
+the anchor character stays fixed and the cursor moves around.
+.Pp
+The options are as follows:
+.Bl -tag -width indent
+.It Fl help
+Display a help message and quit.
+.
+.It Fl version
+Display Kakoune version and quit.
+.
+.It Fl n
+Do not load the system's kakrc.
+.Pq often, Pa /usr/share/kak/kakrc
+.
+.It Fl l
+List existing sessions.
+.
+.It Fl d
+Run as a headless session
+.Pq requires Fl s .
+.
+.It Fl e Ar command
+Execute
+.Ar command
+after the client initialization phase.
+.
+.It Fl E Ar command
+Execute
+.Ar command
+after the server initialization phase.
+.
+.It Fl f Ar keys
+Enter in
+.Sq filter mode :
+select the whole file, then execute
+.Ar keys .
+.
+.It Fl i Ar suffix
+Backup the files on which a filter is applied, using the given suffix.
+.
+.It Fl q
+When in
+.Sq filter mode ,
+don't print any errors
+.
+.It Fl p Ar session_id
+Send the commands written on the standard input to session
+.Ar session_id .
+.
+.It Fl c Ar session_id
+Connect to the given session
+.Ar session_id .
+.
+.It Fl s Ar session_id
+Set the current session name to
+.Ar session_id .
+.
+.It Fl ui Ar type
+Select user interface
+.Ar type ,
+which can be
+.Em ncurses ,
+.Em dummy ,
+or
+.Em json .
+.
+.It Fl clear
+Remove sessions that were terminated in an incorrect state
+.Pq e.g. after a crash .
+.
+.It Fl ro
+Begin in
+.Em readonly mode ,
+all the buffers opened will not be written to disk.
+.
+.It Ar +line Ns Oo Ar :column Oc | Ar +:
```

Comment:
This should be `.It Sy + Ns Ar line Ns Oo Sy \&: Ns Ar column Oc | Sy +:`. See above comment.